src/main/java/Duke.java

```diff
@@ -1,10 +1,69 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    public static void printLine() {
+        System.out.println("______________________________________________________________");
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line;
+        String lowerCaseLine;
+        Task[] textList = new Task[100]; //store in array of Tasks to track both description and mark status
```

Comment:
Perhaps you can create a variable for the array size 100? So that it will be easier to change in the future.